frontend/src/Pages/Control/control.tsx

```diff
@@ -39,6 +39,19 @@ const useStyles = createUseStyles({
   block: {
     flexGrow: "1",
   },
+  logoPreview: {
+    maxWidth: "200px",
+    maxHeight: "200px",
```

Comment:
If you want a consistent size id suggest using em or rem instead of px as rem/em will take the device into account